src/containers/dashboard/MapInfoDistrict.js

```diff
@@ -16,89 +16,143 @@ class DistrictsMapInfo extends Component {
+                        <td>
+                          {key === "icu_trained_nurses"
+                            ? "Adequate Staffing"
+                            : key
+                                .replace(
+                                  /\w\S*/g,
+                                  (txt) =>
+                                    txt.charAt(0).toUpperCase() +
+                                    txt.substr(1).toLowerCase()
+                                )
+                                .replace(/_/g, " ")}
+                        </td>
+                        <td>{val}</td>
+                      </tr>
+                    )
+                  )}
+                </tbody>
+              </table>
+            </div>
+          </React.Fragment>
+        ) : categoryDisplay === 1 && currentDistrict ? (
+          <React.Fragment>
+            <div className="MapInfo_Section">
+              <h3 className="MapInfo_Header color-dark-blue larger-text">
+                District {currentDistrict.district}
+              </h3>
 
-						<div className="MapInfo_Section">
-							<div className="color-light-gray">Reports citing not tested</div>
+              <h4 className="MapInfo_Header color-light-gray">Date Range</h4>
+              <div className="color-dark-blue small-text">
+                {formatDateData(firstReportDate)} -{" "}
+                {formatDateData(requestedReport.reportedDate)}
+              </div>
+            </div>
+            <div className="MapInfo_Section">
+              <h4 className="MapInfo_Header color-light-gray">Total Reports</h4>
 
-							<div className="color-dark-blue larger-text">{allTotals.citedNoTesting}</div>
-						</div>
+              <div className="color-dark-blue larger-text">
+                {allTotals.reports}
+              </div>
+            </div>
 
-						<div className="MapInfo_Section">
-							<div className="color-dark-blue medium-text">Testing Status</div>
-							<div>
-								<div>Haven't sought: {allTotals.testingTableData.testNone}</div>
-								<div>Testing unavailable: {allTotals.testingTableData.testTried}</div>
-								<div>Tested - result pending: {allTotals.testingTableData.testNoResult}</div>
-								<div>Tested - has result: {allTotals.testingTableData.testResults}</div>
-							</div>
-						</div>
-					</React.Fragment>
-				) : (
-					<div className="MapInfo_Section">
-						<div className="color-dark-blue larger-text">District</div>
-						<div className="color-light-gray">Select a district</div>
-					</div>
-				)}
-			</div>
-		);
+            <div className="MapInfo_Section">
+              <h4 className="MapInfo_Header color-light-gray">
+                Reports citing not tested
+              </h4>
+
+              <div className="color-dark-blue larger-text">
+                {allTotals.citedNoTesting}
+              </div>
+            </div>
+
+            <div className="MapInfo_Section">
+              <table>
```

Comment:
Interesting change, what's the motivation?

Reply:
The tables still weren't styled yet and since they're tables, I figured they should be semantically as well.